salt/modules/purefa.py

```diff
@@ -27,6 +27,17 @@
 
       pip install purestorage
 
+- Configure Pure Storage FlashArray authentication. Use one of the following
+  three methods.
+
+    1) From the minion config
+        pure_tags:
```

Comment:
I think it would be useful for this block to be code-formatted. Can you change this to:
```
1) From the minion config
    .. code-block:: yaml

        pure_tags:
          fa: 
...
2) From environment...
```

Reply:
@rallytime Good point. Fixed. Thanks.